docs/UserGuide.md

```diff
@@ -230,15 +235,55 @@ Deletes the stock(s) using the stock's serial number from the inventory. Multipl
 <h4>Format</h4>
 
 ```
-delete sn/<serial number>...
+delete sn/<serial number> [sn/<serial number>]...
```

Comment:
I think `sn/<serial number>...` is already correct.

Reply:
I think its better to tell user that ... refers to zero or more sn/ fields. They might think ... should mean anything else at one glance.